packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -956,7 +960,10 @@ export class DashboardModel {
                 '=',
                 `${UserTableName}.user_uuid`,
             )
-            .select(
+            .select<
+                (DbDashboardTileComments &
```

Comment:
since it joins another table the types don't come through _easily_ - so I checked other parts of the code where something similar to this is done